styles/styles.css

```diff
@@ -0,0 +1,336 @@
+body {
+  margin: 0;
+  width: 100%;
+}
+
+/*
+header {
+  background-color: #0A3C61;
+  height: 21rem;
+  text-align: left;
+  display: flex;
+  justify-content: center;
+}
+*/
+header {
+  background-color: 10, 11, 12 ;
+  height: 24rem;
+  justify-content: flex-start;
+
+}
+
+header h1 {
+  color: black;
+  position: absolute;
+  left: 13rem;
+  top: 1rem;
+  font-size: 60px;
+  font-family: sans-serif;
+}
+
+header h2 {
+  color: white;
+  position: absolute;
+  font-size: 25px;
+  font-family: sans-serif;
+  font-style: italic;
+  line-height: 36px;
+}
+.header-images {
+  width: 100%;
+  height: 24rem;
+  background-image: url(../images/header-image.jpg);
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+
+.b-box {
+  width: 30rem;
+  height: 100px;
+}
+
+
+.header-img {
+  object-fit: cover;
+  width: 100%;
+  height: inherit;
+  object-position: top;
+}
+
+
+
+.ftrlogo {
+  height: 30%;
+}
+
+.background2Box {
+  height: 70%;
+  width: 40%;
+  margin: 4%;
+  min-width: 227px;
+  max-width: 484px;
+}
+
+.headertext  </div> {
+  justify-content: center;
+}
+
+.checkout {
+  margin: 4%;
+}
+
+.art1 {
+  width: 60%;
+  display: inline-block;
+  margin: 10%;
+}
+
+nav {
+  font-family: sans-serif;
+}
+
+nav li {
+  padding: 0 5px 0 5px;
+  position: relative;
+  left: 11rem;
+}
+
+nav li a:after{
+  content: " / ";
+  position: absolute;
+  right: 0;
+  top : 15px;
+}
+
+nav li:last-child a:after{
+    display: none;
+}
+
+nav ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  height: 50px;
+  overflow: hidden;
+  background-color: rgb(235, 235, 235);
+}
+
+nav li {
+  float: left;
+}
+
+nav li a {
+  display: block;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+nav li a:hover {
+  background-color: Grey;
+}
+
+#wrapper {
+  width: 80%;
+  min-height: auto;
+  margin: 2rem auto;
+  overflow: hidden;
+  line-height: 32px;
+  font-size: 16px;
+  font-family: sans-serif;
+}
+
+.flex-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.flex-container > div {
+  background-color: #f1f1f1;
+  margin: 10px;
+  text-align: center;
+  line-height: 75px;
+  font-size: 30px;
+}
+
+.dogimgages {
+  width: 100%;
+}
+
+.div1 {
+  width: 98%;
+  display: inline-table;
+}
+
+.div2 {
+  width: 22%;
+  min-width: 275px;
+}
+
+.col {
+    width: 90%;
+    margin: 2rem auto;
+    float: left;
+    padding: 5px;
+}
+
+.row::after {
+    content: "";
+    clear: both;
+    display: table;
+}
+
+.span_3_of_12 {
+  width: 25%;
+}
+
+.span_8_of_12 {
+  width: 66.66%;
+}
+
+.span_9_of_12 {
+  width: 75%;
+}
+
+
+
+.hide {
+  display: none;
+}
+
+.left {
```

Comment:
avoid float. Use a more modern layout technique. Flexbox is a better option for this